junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileSource.java

```diff
@@ -94,4 +94,14 @@
 	@API(status = EXPERIMENTAL, since = "5.5")
 	String emptyValue() default "";
 
+	/**
+	 * A List of Strings that should be interpreted as {@code null}.
+	 *
+	 * <p>Defaults to {@code {}}.
+	 *
+	 * @since 5.5
```

Comment:
5.5 has already been released, so please update to 5.6 accordingly.

Reply:
The version is adjusted.
Also done in the `CsvSource`

Reply:
Thanks